src/test/java/org/tarantool/ClientReconnectClusterIT.java

```diff
@@ -32,48 +33,41 @@
         "return box.schema.space.create('%1$s').id, box.space.%1$s:create_index('primary').id";
 
     private static final int TIMEOUT = 500;
-    private static final String LUA_FILE = "jdk-testing.lua";
-    private static final String SRV1 = "replica1";
-    private static final String SRV2 = "replica2";
-    private static final String SRV3 = "replica3";
+
+    private static final String SRV1 = "cluster-replica1-it";
+    private static final String SRV2 = "cluster-replica2-it";
+    private static final String SRV3 = "cluster-replica3-it";
     private static final int[] PORTS = { 3401, 3402, 3403 };
     private static final int[] CONSOLE_PORTS = { 3501, 3502, 3503 };
-    private static TarantoolControl control;
+
+    // Resume replication faster in case of temporary failure to fit TIMEOUT.
+    private static double REPLICATION_TIMEOUT = 0.1;
 
     private static String REPLICATION_CONFIG = TestUtils.makeReplicationString(
-        AbstractTarantoolConnectorIT.username,
-        AbstractTarantoolConnectorIT.password,
+        TarantoolTestHelper.USERNAME,
+        TarantoolTestHelper.PASSWORD,
         "localhost:" + PORTS[0],
         "localhost:" + PORTS[1],
         "localhost:" + PORTS[2]
     );
 
-    // Resume replication faster in case of temporary failure to fit TIMEOUT.
-    private static double REPLICATION_TIMEOUT = 0.1;
+    private static Map<String, TarantoolTestHelper> instances = new HashMap<>();
 
     @BeforeAll
     public static void setupEnv() {
-        control = new TarantoolControl();
-        int idx = 0;
+        int index = 0;
         for (String name : Arrays.asList(SRV1, SRV2, SRV3)) {
-            control.createInstance(name, LUA_FILE,
-                makeInstanceEnv(
-                    PORTS[idx], CONSOLE_PORTS[idx],
-                    REPLICATION_CONFIG, REPLICATION_TIMEOUT
-                )
+            TarantoolTestHelper testHelper = new TarantoolTestHelper(name);
+            testHelper.createInstance(
+                TarantoolTestHelper.LUA_FILE,
+                PORTS[index],
+                CONSOLE_PORTS[index],
+                REPLICATION_CONFIG,
+                REPLICATION_TIMEOUT
             );
-            idx++;
-        }
-    }
+            instances.put(name, testHelper);
 
-    @AfterAll
-    public static void tearDownEnv() {
```

Comment:
Do we stop instances after all tests?

Reply:
yes, we do stop after each test case [here](https://github.com/tarantool/tarantool-java/blob/d057977d0a3bd09a31cd65eff8b06ed8073e2a51/src/test/java/org/tarantool/ClientReconnectClusterIT.java#L81)